website/docs/providers/entra/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Entra ID provider
+---
+
+<span class="badge badge--primary">Enterprise</span>
+
+---
+
+With the Microsoft Entra ID provider, authentik can be the single source of truth for all users and groups. Configuring Entra ID as a provider allows for auto-discovery of user and group accounts, on-going synching of user data such as email address, name, and status, and integrated data mapping of field names and values.
+
+-   For instructions to configure your Entra ID system to integrate with authentik, refer to [Configure Entra ID](./setup-entra).
+-   For instructions to add Entra ID as a provider in authentik, refer to [Create a Entra ID provider](./add-entra-provider).
+
+## About using Entra ID with authentik
+
+The following sections discuss how Entra ID operates with authentik.
+
+### Discovery
+
+When first creating the provider and setting it up correctly, the provider will run a discovery and query your Entra ID for all users and groups, and attempt to match them with their respective counterparts in authentik. This discovery takes into consideration any **User filtering** options configured in the provider, such as only linking to authentik users in a specific group or excluding service accounts.
+
+This discovery happens every time the provider is saved.
+
+### Syncing
+
+There are two types of sync: a direct sync and a full sync.
+
+The full sync happens when the provider is initially created and when it is saved. The full sync goes through all users and groups matching the **User filtering** options set and will create/update them in Entra ID. In addition to that, the full sync is also run on a schedule every 4 hours.
+
+The direct sync happens when a user or a group is created/updated/deleted in authentik, or a member is added/removed to/from a group. The direct sync will only forward those changes to Entra ID.
+
+During either sync, if a user or group was created in authentik and a matching user/group exists in Entra ID, authentik will link them together as the discovery would do above.
```

Comment:
"... authentik will automatically link them together."  I think the rest is distracting fluff, asking the reader to recall something above about _process_ rather than being clear about the intent of this sentence, which is the _consequence_.